src/sentry/static/sentry/app/views/settings/organization/apiKeys/organizationApiKeyDetailsView.jsx

```diff
@@ -52,11 +52,20 @@ class OrganizationApiKeyDetailsView extends OrganizationSettingsView {
   }
 
   handleSubmitSuccess = () => {
-    IndicatorStore.add(t('Saved changes'), 'success');
+    addSuccessMessage('Saved changes');
+
+    // Go back to API list
+    browserHistory.push(
```

Comment:
We have a mixed approach to this throughout the app right now. In general I feel that when we save any resource we should stay on the page and not navigate away, getting taken to the collection view feels unexpected to me

Reply:
Yeah I think generally we bring you back to the list view. I think it makes sense since you click save, you're done editing the resource.

Reply:
Not a blocker, just my 2c